### DISCUSSION TITLE
bash: line 16: cargo: command not found

### DISCUSSION THREAD

**Discussion by Ex-8** [Drew-Alleman/DataSurgeon#5] [Q&A] [answered]
Hello here I am getting an error How can I solve it

──(kali㉿kali)-[~/Desktop]
└─$ wget -O - https://raw.githubusercontent.com/Drew-Alleman/DataSurgeon/main/install/install.sh | bash

--2023-04-05 22:27:31--  https://raw.githubusercontent.com/Drew-Alleman/DataSurgeon/main/install/install.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.110.133, 185.199.108.133, 185.199.109.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.110.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 752 [text/plain]
Saving to: ‘STDOUT’

-                                                          100%[=======================================================================================================================================>]     752  --.-KB/s    in 0s      

2023-04-05 22:27:31 (3.00 MB/s) - written to stdout [752/752]

[*] Downloading DataSurgeon's source from Github....
bash: line 16: cargo: command not found

**Comment by Drew-Alleman** [comment, marked as answer]
Looks like rust isn't installed. You can follow the video guide on my readme to install rust.

**Comment by Ex-8** [comment]
thank you problem solved
